code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
+        <h2>Broccoli 🥦</h2>
+        <p>A humble and delicious vegetable</p>
+        <p>Here are my favourite things about it</p>
+        <ol class='list'>
+            <li>Tastes Great</li>
```

Comment:
What do you think about doing 3 things instead? As they say, all good things come in 3 :)